src/bundle/Controller/ContentOnTheFlyController.php

```diff
@@ -354,6 +363,9 @@ private function buildEditView(
             'language' => $language,
             'content_type' => $contentType,
             'form' => $form->createView(),
+            'grouped_fields' => $this->groupedContentFormFieldsProvider->getGroupedFields(
+                $form->get('fieldsData')->all()
+            ),
```

Comment:
My only note is here, that field grouping should be done in the form type itself, during `buildView` method.